sci-libs/cminpack/cminpack-1.3.8.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="C implementation of the MINPACK nonlinear optimization library"
+HOMEPAGE="http://devernay.free.fr/hacks/cminpack/"
+SRC_URI="https://github.com/devernay/cminpack/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="minpack"
+SLOT="0/1"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc test"
+RESTRICT="!test? ( test )"
+
+src_configure() {
+	local mycmakeargs=(
```

Comment:
Please check out my bump of this version in commit 71c501b2822cffd4da67aa754925f7ce6d002288, you were missing a new default enabled option here.

I also had to patch CMakeLists.txt to make it install its cmake config into the usual path.

Either drop the bump or rebase and work on adding a new `IUSE` here (I just disabled it for now).

Reply:
Thanks, I missed it somehow. I'll just drop this commit we can revisit blas later